core/src/main/java/org/apache/carbondata/core/metadata/converter/ThriftWrapperSchemaConverterImpl.java

```diff
@@ -219,6 +219,10 @@
     externalPartitionInfo.setList_info(wrapperPartitionInfo.getListInfo());
     externalPartitionInfo.setRange_info(wrapperPartitionInfo.getRangeInfo());
     externalPartitionInfo.setNum_partitions(wrapperPartitionInfo.getNumPartitions());
+    externalPartitionInfo.setNumOfPartitions(wrapperPartitionInfo.getNumberOfPartitions());
+    externalPartitionInfo.setMax_partition(wrapperPartitionInfo.getMAX_PARTITION());
+    externalPartitionInfo.setTaskId_In_PartitionOrder(wrapperPartitionInfo
```

Comment:
TaskIdInPartitionOrder is an array used to ensure taskId could mapping to correct partition after alter table partition.
It will be used when alter partition table. (The operation method will be uploaded with alter table PR)